web/src/components/storage/BootConfigField.test.tsx

```diff
@@ -54,48 +47,18 @@ const sda = {
   udevPaths: ["pci-0000:00-12", "pci-0000:00-12-ata"],
 };
 
-/** @type {BootConfigFieldProps} */
-let props;
+const props: BootConfigFieldProps = {
+  configureBoot: false,
+  bootDevice: undefined,
+  defaultBootDevice: undefined,
+  availableDevices: [sda],
+  isLoading: false,
+};
 
 beforeEach(() => {
-  props = {
-    configureBoot: false,
-    bootDevice: undefined,
-    defaultBootDevice: undefined,
-    availableDevices: [sda],
-    isLoading: false,
-    onChange: jest.fn(),
-  };
 });
 
-/**
- * Helper function that implicitly test that field provides a button for
- * opening the dialog
- */
-const openBootConfigDialog = async () => {
-  const { user } = plainRender(<BootConfigField {...props} />);
-  const button = screen.getByRole("button");
-  await user.click(button);
-  const dialog = screen.getByRole("dialog", { name: "Partitions for booting" });
-
-  return { user, dialog };
-};
-
```

Comment:
NP: Should we take the opportunity to drop the below `.skip`?

Reply:
I tried, but the `Link` we use in the component gave me some problems. So I decided to postpone it.